Aligner/src/main/java/edu/stanford/hivdb/alignment/NucAminoAligner.java

```diff
@@ -474,16 +474,15 @@ private static AlignedGeneSeq geneSeqFromReport(
 			executable = "nucamino";
 		}
 		return new String[] {
-			executable,
-			"hiv1b", // hiv1b sub-command
-			"-q", // quiet mode
-			"--gene", "POL", // align input sequence with POL consensus
-			"--output-format", "json",
-			"--indel-codon-opening-bonus", "" + INDEL_CODON_OPENING_BONUS,
```

Comment:
Please also remove the unused constants.

Reply:
Updated NucAmino to v0.2.0-beta-2; removed unused constants (thank you for catching that); passed tests via ./gradlew test

Reply:
(NucAmino updated in .travis.yml)

Reply:
spotted error. fixing now.